src/scaffoldmaker/annotation/bladder_terms.py

```diff
@@ -4,14 +4,16 @@
 
 # convention: preferred name, preferred id, followed by any other ids and alternative names
 bladder_terms = [
-    ("urinary bladder", "FMA:15900", "UBERON:0001255"),
-    ("neck of urinary bladder", "FMA:15912", "UBERON:0001258"),
+    ("urinary bladder", "UBERON:0001255", "FMA:15900"),
+    ("neck of urinary bladder", "UBERON:0001258", "FMA:15912"),
     ("Dome of the Bladder", "ILX:0738433"),
+    ("serosa of urinary bladder", "UBERON:0001260"),
+    ("bladder lumen", "UBERON:0009958"),
     ("serosa of body of urinary bladder", "ILX:0739276"),
     ("lumen of body of urinary bladder", "ILX:0739252"),
     ("serosa of neck of urinary bladder", "ILX:0739277"),
     ("lumen of neck of urinary bladder", "ILX:0739256"),
-    ("urethra", "ILX:0733022"),
+    ("urethra", "ILX:0733022", "UBERON:0000057"),
```

Comment:
Normally prefer to use the UBERON term ahead of ILX; the first one in the list after the name is always used (the rest are there in case we need to match other data). This is unless you have to match digitized contours which are annotated with the ILX term.